components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -819,19 +834,40 @@ supervisorLoop:
 		startTime := time.Now()
 		cmd = prepareIDELaunch(cfg, ideConfig)
 		launchIDE(cfg, ideConfig, cmd, ideStopped, ideReady, &ideStatus, ide)
+		timer10s := time.NewTimer(11 * time.Second)
```

Comment:
can we use some shared constant for max bucket + 1? or add a comment which links places?